backends/p4test/CMakeLists.txt

```diff
@@ -108,6 +108,9 @@ set (P4_XFAIL_TESTS
   # These programs uses conditional execution inside action body which is currently
   # unsupported
   testdata/p4_16_samples/psa-dpdk-action-jumpOpt.p4
+  testdata/p4_16_samples/pna-example-tcp-connection-tracking.p4
```

Comment:
Done.

Reply:
I think I misunderstood this comment and made the recent changes to midend/predication.h by removing the check for conditional execution inside action body.

Some of the tests added in this PR fail with p4test backend because of use of conditional execution within action. So, I added them to XFAIL list for p4test.

Shall I revert back to that, .i.e., no change to the predication.h and adding back the tests with conditional execution in action as XFAIL here?